istioctl/cmd/analyze.go

```diff
@@ -288,6 +306,9 @@ istioctl analyze -L
 		"Overrides the mesh config values to use for analysis.")
 	analysisCmd.PersistentFlags().BoolVar(&allNamespaces, "all-namespaces", false,
 		"Analyze all namespaces")
+	analysisCmd.PersistentFlags().StringArrayVar(&suppress, "suppress", []string{},
+		"Suppress reporting a message code on a specific resource. Values are supplied in the form "+
+			"<code>=<resource> (e.g. '--suppress \"IST0102=namespace default\"'). Can be repeated.")
```

Comment:
Maybe use an example that includes the full `<kind> <name>.<namespace>` format? If we're going to just include one example, that seems the most useful.

Reply:
Good call. Updated.